src/client/service-worker/events/utility.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import {
+  match, put, remove, getMetaData, setMetaData, createCacheName,
+} from '@americanexpress/one-service-worker';
+
+const localeRegExp = /(?<locale>([a-z]{2,3})(?:-)?([a-zA-Z]{1,})?)\/[^/]*\.json/;
+
+export function createResourceMetaData(event, resourceInfo, revision) {
+  const [name, baseUrl] = resourceInfo;
+  const [version] = baseUrl.replace(/\/$/, '').split('/').reverse();
+  const { request } = event;
+
+  let type = name === 'app' ? 'one-app' : 'modules';
+
+  let locale = null;
```

Comment:
why just not leave it as ? 
```suggestion
  let locale;
```